docs/src/dev/developer/development-environment.asciidoc

```diff
@@ -503,14 +511,16 @@ To be a bit more clear, the version can not be a Docker tag like "latest" becaus
 that has been published with that version number.
 
 [[docker-testing]]
-== Testing with Docker
+== Testing GLVs with Docker
 
-Currently gremlin-go, gremlin-javascript, gremlin-dotnet, gremlin-python and gremlin-console can be tested through Docker.
+Currently gremlin-go, gremlin-javascript, gremlin-dotnet, gremlin-python and gremlin-console can be tested through Docker using
```

Comment:
gremlin-console actually just uses a regular Dockerfile now, and not compose.

Reply:
Updated